st2auth/conf/apache.full.conf

```diff
@@ -0,0 +1,49 @@
+<VirtualHost *:443>
+
+    ServerName 172.168.50.50:443
```

Comment:
We should just leave ServerName out.  If we want it to only listen on that IP, then it should be up in the VirtualHost definition instead of *